content/css/concepts/cursor/cursor.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: 'cursor'
+Description: 'Sets the appearance of the mouse cursor when hovering over an element.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'UI'
+CatalogContent:
+  - 'learn-css'
+  - 'learn-intermediate-css'
+---
+
+The `cursor` property sets the appearance of the mouse cursor when hovering over an element. The appearance can be set by specifying the type of cursor, or by loading a specific icon. When using an icon, fallback images can be used, ending with a mandatory keyword fallback.
+
+These are common keywords used to specify the type of cursor to display:
+
+![Cursors](https://raw.githubusercontent.com/Codecademy/docs/main/media/cursors-1.png)
```

Comment:
I don't think it would wok with the current graphic. I could replace it with a bulleted list, but I thought the images were more helpful.

Reply:
Maybe we could combine these two ideas into a markdown table?

Example:

| Name | Image | Description |
| ------ | ------ | ----------- |
| Default | ![default-cursor alt-text](...) | Used on most platforms; looks like an arrow. |
| Progress | ![progress-cursor alt-text](...) | Running in the background; user actions allowed. |
| Wait | ![wait-cursor alt-text](...) | Running in the background; user actions not allowed. |

Reply:
Maybe instead of explaining each one, we can group certain ones together depending on their purpose?

Like which ones are used for selections, text, etc. Any rather than having those explanations in the images, we can include some bullet points underneath the image 🤔

Reply:
I like the markdown table, too. To incorporate some of what @yangc95, the keywords could be grouped by category within the table. Similar to how they appear to be listed by category in the image.

Reply:
I'll try the markdown table approach. However, it will spam the media folder with a bunch of images for one entry. Anyone see this as a problem?

Reply:
😮  the image size is rather small so I don't think it would be an issue